Marlin/src/lcd/menu/menu_item.h

```diff
@@ -342,8 +342,8 @@ class MenuItem_bool : public MenuEditItemBase {
 
 #define PSTRING_ITEM(LABEL, V...)                     PSTRING_ITEM_P(GET_TEXT(LABEL), ##V)
 
-#define STATIC_ITEM(LABEL,      V...)                  STATIC_ITEM_P(GET_TEXT(LABEL), ##V)
-#define STATIC_ITEM_N(LABEL, N, V...)                STATIC_ITEM_N_P(GET_TEXT(LABEL), N, ##V)
+#define STATIC_ITEM(LABEL,   V...)                     STATIC_ITEM_P(GET_TEXT(LABEL), ##V)
+#define STATIC_ITEM_N(LABEL, V...)                   STATIC_ITEM_N_P(GET_TEXT(LABEL), ##V)
```

Comment:
im confused about the removal of the N in the `#define STATIC_ITEM_N(LABEL, V...)                   STATIC_ITEM_N_P(GET_TEXT(LABEL), ##V)` because its looking for it, right?
```
 #define STATIC_ITEM_N_P(PLABEL, N, V...) do{ \
  if (_menuLineNr == _thisItemNr) {          \
    MenuItemBase::init(N);                   \
    STATIC_ITEM_INNER_P(PLABEL, ##V);        \
  }                                          \
  NEXT_ITEM();                               \
}while(0)
```